pyproject.toml

```diff
@@ -19,6 +19,7 @@ google-api-python-client = "^1.10.0"
 google-auth-httplib2 = "^0.0.4"
 google-auth-oauthlib = "^0.4.1"
 oauth2client = "^4.1.3"
+prometheus_client = "^0.8.0"
```

Comment:
make sure to add `'prometheus-client==0.8.0'` to line `23` on setup.py and run `make gen_requirements` and `make gen_requirements_dev` to update requirements.txt and requirements-dev.txt